Doc/search_scorer.js

```diff
@@ -0,0 +1,24 @@
+var Scorer = {
+  score: function (result) {
+    let [docname, title, anchor, descr, score, filename] = result;
+    if (docname == "library/stdtypes" || docname == "library/functions") {
+      score += 10;
+    }
+    return score;
+  },
+
+  objPrio: {
```

Comment:
Perhaps, you could add comments for different options, this one looks especially cryptic for me.

Reply:
Can do.